crates/miden-tx/src/auth/tx_authenticator.rs

```diff
@@ -35,8 +75,7 @@ pub trait TransactionAuthenticator {
     fn get_signature(
         &self,
         pub_key: Word,
```

Comment:
Not your code, but while we're at it, can we rename this (here and where the trait is implemented) to `pub_key_hash` to make it clear that it's the hash of a public key and not the public key itself?

Ideally also update the doc above:

```
/// - `pub_key_hash`: The hash of the public key used for signature generation.
```

If you don't want to do it here, let me know and I'll create an issue for it.

Reply:
https://github.com/0xMiden/miden-base/pull/1624#discussion_r2222273024 😆

Reply:
Done in 1aedd73a